### ISSUE TITLE
Add a skip link

### ISSUE THREAD

**Issue by carolinekrone** [laboratoriobridge/bold#396]
Add a skip link to the main content of each page. 

WCAG rule: [2.4.1 - Bypass Blocks / G1 ](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G1)

**Comment by CarlosBonetti** [comment]
Should be implemented as a project-specific solution. Not a design system solution.

**Comment by CarlosBonetti** [comment]
Fixed by f29dc03598d42b2a3a1f330f84c5b78d16d4a938
